components/script/dom/htmliframeelement.rs

```diff
@@ -73,6 +83,7 @@ pub struct HTMLIFrameElement {
     sandbox_allowance: Cell<Option<SandboxAllowance>>,
     load_blocker: DOMRefCell<Option<LoadBlocker>>,
     visibility: Cell<bool>,
+    nested_browsing_context: MutNullableHeap<JS<BrowsingContext>>,
```

Comment:
It's not clear to me what the purpose of this is, except for asserting that it doesn't exist in create_nested_browsing_context.